toolkits/landcover/impl/bands.js

```diff
@@ -2,22 +2,22 @@
  * @license
  * Copyright 2019 Google LLC
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 2.0 (the 'License');
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  *    https://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
+ * distributed under the License is distributed on an 'AS IS' BASIS,
```

Comment:
Looks like you may have done a search & replace instead of `npm run fix`? Please revert `"` in docs to original state.

Reply:
I don't see any new commits.. did you forget to push?